sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientLiveTests.cs

```diff
@@ -707,7 +707,8 @@ public async Task CanSetServiceTimeout()
                     // or a partial failure 200 response
                     if (exception.Status == 200)
                     {
-                        StringAssert.Contains("Query cancelled by the user's request", exception.Message);
+                        //StringAssert.Contains("Query cancelled by the user's request", exception.Message);
```

Comment:
TODO: confirm before merging that error message comparison is okay

Reply:
Partial failure message has just changed so I think this is fine as it still checks for the same error.